packages/gatsby-source-vtex/src/gatsby-node.ts

```diff
@@ -282,6 +282,18 @@ export const createPages = async (
     },
   })
 
+  // Use checkout IO
+  createRedirect({
+    fromPath: '/cart/*',
+    toPath: `https://${tenant}.${environment}.com.br/cart/:splat`,
```

Comment:
Shouldn't we use the `{workspace}--myvtex.com` host since this is the io checkout ?